front/src/applications/operationalStudies/views/Project.tsx

```diff
@@ -277,6 +331,39 @@ export default function Project() {
                 id="studies-filter"
               />
             </div>
+            {selectedStudieIds.length > 0 && (
+              <>
+                <AiFillCheckCircle />
+                <span className="ml-0">
+                  {t('selectedProjects', { count: selectedStudieIds.length })}
```

Comment:
I think 'selectedScenarios' would be more appropriate.

```suggestion
                  {t('selectedStudies', { count: selectedStudieIds.length })}
```

Reply:
Done 👍